From 076ba1ccc04b7478988bed0283c1dcf358d9d9a3 Mon Sep 17 00:00:00 2001
From: Michael Haubenwallner <michael.haubenwallner@ssi-schaefer.com>
Date: Tue, 21 May 2019 17:57:29 +0200
Subject: [PATCH 07/19] link libmico.so against detected libraries

Link shared libmico.so against configure detected libraries to avoid
undefined symbols, to let --as-needed linker flag work if forced.
See also: https://bugs.gentoo.org/280678
---
 orb/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/orb/Makefile b/orb/Makefile
index 2bf0627..39e8e93 100644
--- a/orb/Makefile
+++ b/orb/Makefile
@@ -30,7 +30,7 @@ CXXFLAGS := -DBUILD_MICO_DLL $(CXXFLAGS)
 endif
 endif
 
-PLATFORM_LIBS =
+PLATFORM_LIBS = $(CONFLIBS)
 ifeq ($(HAVE_CYGWIN), yes)
 ifeq ($(USE_CSL2), yes)
 PLATFORM_LIBS := $(PLATFORM_LIBS) -lssl -lcrypto
-- 
2.19.2

